File: backend/controller/marker.js
const knex = require('../knexfile');

module.exports = {
    async list() {
        const marker = knex('marker')
            .select('latitude', 'longitude')
            .map(r => ({
                latitude: r.latitude,
                longitude: r.longitude
            }))
        return marker;
    }
};

File: frontend/app/src/main/java/com/example/footprint/model/Marker.java
package com.example.footprint.model;

import org.json.JSONException;
import org.json.JSONObject;

public class Marker {
    private String latitude;
    private String longitude;
    private Integer type;

    public Marker(JSONObject jsonObject) {
        try {
            this.latitude = (String) jsonObject.get("latitude");
            this.longitude = (String) jsonObject.get("longitude");
            this.type = (Integer) jsonObject.get("type");
        } catch (JSONException e) {
            e.printStackTrace();
        }
    }

    public String getLatitude() {
        return latitude;
    }

    public String getLongitude() {
        return longitude;
    }

    public Integer getType() {
        return type;
    }
}
